text/src/autogluon/text/text_prediction/presets.py

```diff
@@ -0,0 +1,214 @@
+__all__ = ['ag_text_presets', 'list_presets']
+
+import copy
+import logging
+import functools
+from autogluon_contrib_nlp.utils.registry import Registry
+from autogluon.core import space
+
+logger = logging.getLogger(__name__)  # return root logger
+
+ag_text_presets = Registry('ag_text_presets')
+
+
+def list_presets():
+    """List the presets available in AutoGluon-Text"""
+    return ag_text_presets.list_keys()
+
+
+def base() -> dict:
+    """The default hyperparameters.
+
+    It will have a version key and a list of candidate models.
+    Each model has its own search space inside.
+    """
+    ret = {
+        'version': 1,                     # Version of TextPrediction Model
```

Comment:
Do we really need this `version`, isn't it tied to the autogluon version already?
If it's really needed, I think you may want to use more informative names rather than simply version 1, 2, ...

Reply:
Nice catch. I removed this version flag.